controller/controller_server/exception_handler.py

```diff
@@ -35,15 +33,3 @@ def build_error_response(message, context, status_code, response_type):
 def handle_exception(ex, context, status_code, response_type):
     logger.exception(ex)
     return _build_non_ok_response(str(ex), context, status_code, response_type)
-
-
-def handle_common_exceptions(response_type):
-    @decorator
-    def handle_common_exceptions_with_response(controller_method, servicer, request, context):
-        try:
-            return controller_method(servicer, request, context)
-        except Exception as ex:
-            status_code = status_codes_by_exception.get(type(ex), grpc.StatusCode.INTERNAL)
-            return handle_exception(ex, context, status_code, response_type)
```

Comment:
I think this should stay here (not as a decorator), and be called from `decorators.py`, to reduce the long `csi_method` function, and to not have a `try` in the middle of a function

Reply:
done